runtime.md

```diff
@@ -34,22 +34,23 @@ This is provided so that consumers can find the container's configuration and ro
 ```
 
 ## Lifecycle
-
-### Create
-
-Creates the container: file system, namespaces, cgroups, capabilities.
-
-### Start (process)
-
-Runs a process in a container.
-Can be invoked several times.
-
-### Stop (process)
-
-Not sure we need that from runc cli.
-Process is killed from the outside.
-
-This event needs to be captured by runc to run onstop event handlers.
+The lifecycle describes the timeline of events that happen from when a container is started to when it is stopped or crashes unexpectedly.
+
+1. OCI compliant runtime is invoked by passing the bundle path as argument.
+2. The container's runtime environment is created.
```

Comment:
On Wed, Oct 28, 2015 at 07:36:30AM -0700, Doug Davis wrote:
> > +2. The container's runtime environment is created.
> 
> Should we add?
> ```
> ...is create according to the configuration specified in the
> 'config.json' file.  This file is only processed (in a read-only
> fashion) during the container's creation processing, and any
> external changes made to this file once the container is created
> will not have any effect on the container.
> ```

It also depends on runtime.json (since #88).  But the rest of that
statement still holds after “the 'config.json' file” → “`config.json`
and `runtime.json`”).